fundamentals-recap/math-primer.md

```diff
@@ -0,0 +1,170 @@
+# Math Primer
+
+## Goal
+
+Problem-solving in programming is almost always centered on _data_. Often times, it's useful to be aware of the following math concepts. Knowing the definition of these concepts can become tools you remember to use in the future.
+
+The goal of this document is to be a resource. We recommend spending your energy on this document reading through it and taking notes on it once, and then reviewing briefly every few months.
+
+This document covers the following:
+
+Thinking about numbers:
+
+1. Decimal Number System
+1. Using _n_ as a Variable in Math
+1. Coordinate Planes
+
+Working with collections of numbers:
+
+1. Maximum and Minimum
+1. Sorting
+1. Average or Mean
+1. Floor and Ceiling Functions
+
+## Decimal Number System
+
+The _decimal number system_ or _base-ten number system_ is the numbering system most humans use in everyday life.
+
+This numbering system includes - ten unique symbols known as digits: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 - the decimal point (floating point/dot) symbol to represent decimal fractions, such as 2.45
+
+### !callout-info
+
+## Origin of Base 10
+
+"Legend says" that most base-ten systems possibly originate from counting on ten fingers.
+
+### !end-callout
+
+## Using _`n`_ as a Variable in Math
+
+When we describe problems, we might want to say, "any number of things."
+
+For example, we might be building an email webapp, and we want to say, "An email can have any number of recipients... It can have 1 recipient, 3, 13, or 50 recipients."
+
+We wouldn't want to say "An email can have infinite recipients," because that just may not be true. However, when we want to discuss numbers of things, we may start to use variables as we talk.
+
+We can imagine a variable _`n`_ as the name of a variable that holds any number.
```

Comment:
suggest avoiding invocation of infinity
```suggestion
It would be annoying if we always had to say that an email might have 1, 3, 13, etc., recipients. Instead, we can use a variable to talk about this unknown number.

We can imagine a variable _`n`_ as the name of a variable that holds any number. Then we could say that the email has _`n`_ recipients.
```

Reply:
Thank you for that! (I have a bad habit of using infinity!)